src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -32,26 +31,6 @@ export class Page {
             );
         }
 
-        underlyingPage.on('pageerror', error => {
```

Comment:
Nope, that was an oversight; since that issue for the second bit is still open, we should leave that in. I'll re-add now. Thanks for taking a look at these files